project_sequence/models/project_project.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+
+from odoo import api, fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = "project.project"
+
+    codesequence = fields.Char(
+        readonly=True,
+        copy=False,
+    )
+    name = fields.Char(
+        required=False,
+    )
+    display_name = fields.Char(
+        compute="_compute_display_name_project", store=True, index=True
```

Comment:
You can see that [`display_name` has a compute method enabled by default](https://github.com/odoo/odoo/blob/680ff54c56e3a1c1525999c448385417aa7dbb0f/odoo/models.py#L1715-L1730). You can see there that the method depends on the results of the `name_get` function.

So you don't need to override the compute method here. You just have to override the `name_get()` function, and the rest will be automatic.

You do need to set it as stored and indexed, though:

```suggestion
        store=True, index=True
```